src/compiler/factory/utilities.ts

```diff
@@ -64,7 +64,7 @@ namespace ts {
             );
     }
 
-    export function createExpressionForJsxElement(factory: NodeFactory, jsxFactoryEntity: EntityName | undefined, reactNamespace: string, tagName: Expression, props: Expression | undefined, children: readonly Expression[] | undefined, parentElement: JsxOpeningLikeElement, location: TextRange): LeftHandSideExpression {
+    export function createExpressionForJsxElement(factory: NodeFactory, callee: Expression, tagName: Expression, props: Expression | undefined, children: readonly Expression[] | undefined, location: TextRange): LeftHandSideExpression {

```

Comment:
the signature differs now from the accompanying `createExpressionForJsxFragment` but this is just a util file so I don't think it's particularly important but let me know if you'd like this to be refactored somehow.